File: src/main/java/logan/pickpocket/commands/ProfileCreateCommand.java
package logan.pickpocket.commands;

import logan.api.command.BasicCommand;
import logan.api.command.SenderTarget;
import logan.pickpocket.config.MessageConfiguration;
import logan.pickpocket.main.Profile;
import org.bukkit.command.CommandSender;

import java.util.List;

public class ProfileCreateCommand extends BasicCommand<CommandSender> {

    public ProfileCreateCommand() {
        super("create", "pickpocket.profile.create", 1, 1,
                new String[0], SenderTarget.BOTH, "profile",
                List.of(String.class, String.class),
                "Usage:\n/pickpocket profile create <profile>");
    }

    @Override
    public boolean run(CommandSender sender, String[] args, Object data) {
        boolean result = new Profile(args[0]).save();
        sender.sendMessage(result ?
                MessageConfiguration.getProfileCreateSuccessMessage(args[0]) :
                MessageConfiguration.getProfileErrorAlreadyExistsMessage(args[0]));
        return true;
    }
}

File: src/main/java/logan/pickpocket/commands/ProfileCreateCommand.kt
package logan.pickpocket.commands

import logan.api.command.BasicCommand
import logan.api.command.SenderTarget
import logan.pickpocket.config.MessageConfiguration
import logan.pickpocket.main.Profile
import org.bukkit.command.CommandSender

class ProfileCreateCommand : BasicCommand<CommandSender>(
    "create",
    "pickpocket.profile.create",
    1..1,
    target = SenderTarget.BOTH,
    argTypes = listOf(String::class, String::class),
    parentCommand = "profile",
    usage = """
        Usage:
        /pickpocket profile create <profile>
    """.trimIndent()
) {
    override fun run(sender: CommandSender, args: Array<out String>, data: Any?): Boolean {
        val result = Profile(args[0]).save()
        sender.sendMessage(
            if (result) MessageConfiguration.getProfileCreateSuccessMessage(args[0])
            else MessageConfiguration.getProfileErrorAlreadyExistsMessage(args[0])
        )
        return true
    }
}
